testdata/baselines/reference/submodule/conformance/importEquals3.errors.txt

```diff
@@ -21,16 +20,13 @@ c.ts(3,13): error TS1380: An import alias cannot reference a declaration that wa
     const x = 0;
     export { a, A, x };
     
-==== c.ts (2 errors) ====
+==== c.ts (1 errors) ====
     import * as b from './b';
     import A = b.a.A; // Error
                ~~~~~
 !!! error TS1380: An import alias cannot reference a declaration that was imported using 'import type'.
 !!! related TS1376 b.ts:1:18: 'a' was imported here.
     import AA = b.A; // Error
-                ~~~
-!!! error TS1380: An import alias cannot reference a declaration that was imported using 'import type'.
-!!! related TS1376 b.ts:1:18: 'a' was imported here.

```

Comment:
An error was already reported on the declaration of `A` so this follow-on error isn't necessary.